games-arcade/burgerspace/burgerspace-1.9.2.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A hamburger-smashing video game"
+DESCRIPTION="A game in which you are a chef, and must walk over hamburger in \
```

Comment:
in \n gredients? It looks like you cut the description in the middle of a word here?